kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/consistencycheck/ConsistencyCheckTasksRunner.java

```diff
@@ -99,8 +100,10 @@ protected void runBlocking() {
             DataConsistencyCalculateAlgorithm calculateAlgorithm = jobAPI.buildDataConsistencyCalculateAlgorithm(
                     parentJobConfig, checkJobConfig.getAlgorithmTypeName(), checkJobConfig.getAlgorithmProps());
             this.calculateAlgorithm = calculateAlgorithm;
-            Map<String, DataConsistencyCheckResult> dataConsistencyCheckResult = jobAPI.dataConsistencyCheck(parentJobConfig, calculateAlgorithm);
+            PipelineJobProgressPersistService.addJobProgressPersistContext(checkJobId, jobItemContext.getShardingItem());
```

Comment:
`addJobProgressPersistContext` exists in `ConsistencyCheckJob`, looks it's duplicated